src/services/item/plugins/shortLink/schemas.ts

```diff
@@ -43,57 +46,49 @@ export const getAllByItem = {
   operationId: 'getShortLinksForItem',
   tags: ['shortLink'],
   summary: 'Get all short links for item',
-  description: 'Get all short links information for item.',
+  description:
+    'Get all short links created for an item. The response could be an empty object or a key-value with the platform and the alias.',
 
   params: customType.StrictObject({
     itemId: customType.UUID(),
   }),
-  response: { [StatusCodes.OK]: Type.Array(shortLinkPayload), '4xx': errorSchemaRef },
+  response: {
+    [StatusCodes.OK]: Type.Object({
+      [ShortLinkPlatform.Builder]: Type.Optional(Type.String()),
+      [ShortLinkPlatform.Player]: Type.Optional(Type.String()),
+      [ShortLinkPlatform.Library]: Type.Optional(Type.String()),
+    }),
```

Comment:
The description does not match the schema. 
- Missing returned empty object. If empty does it return with another status code?
- Also does it always return all three platforms?